### PR TITLE
New engine: Genius (lyrics)

### PR DESCRIPTION
**Pull request by woorst**

### CODE REVIEW

**Comment by dalf** [comment]
The travis build shows some errors : 
```
/home/travis/build/asciimoo/searx/tests/unit/engines/test_genius.py:98:121: E501 line too long (125 > 120 characters)
/home/travis/build/asciimoo/searx/tests/unit/engines/test_genius.py:107:121: E501 line too long (127 > 120 characters)
/home/travis/build/asciimoo/searx/tests/unit/engines/test_genius.py:167:121: E501 line too long (122 > 120 characters)
```

you can run these tests on your machine using : 
```sh
./manage tests
```

see https://asciimoo.github.io/searx/dev/quickstart.html

**Comment by kvch** [comment]
I am not sure if it is the fault of the engine or not, but the first few results shows the engine twice.
genius-engine

File: searx/engines/genius.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+from searx.url_utils import urlencode
+from datetime import datetime
+
+# engine dependent config
+categories = ['music']
+paging = True
+language_support = False
+page_size = 5
+
+indicies = ['top_hit', 'song', 'lyric', 'artist', 'album', 'tag', 'video', 'article', 'user']
```
Comment (kvch) [inline searx/engines/genius.py:23]: Why is this list here if it is not used anywhere?

**Review by kvch — CHANGES_REQUESTED** [review]
Please address the review notes.

**Pull request merged**